packages/axway-cli-oum/src/org/usage.js

```diff
@@ -33,15 +35,61 @@ export default {
 		'--to [yyyy-mm-dd]': 'The end date'
 	},
 	async action({ argv, console }) {
-		const { initPlatformAccount } = require('../lib/util');
+		const { initPlatformAccount, formatDate } = require('../lib/util');
 		const { createTable } = require('@axway/amplify-cli-utils');
 		let { account, org, sdk } = await initPlatformAccount(argv.account, argv.org);
-		const { from, to, usage } = await sdk.org.usage(account, org, argv);
+		const { bundle, from, to, usage } = await sdk.org.usage(account, org, argv);
+		const orgEnvs = await sdk.org.environments(account);
+		const maxEntitlement = 9999999999999;
+		const { format } = new Intl.NumberFormat();
+		let bundleValuePadding = 0;
+		let saasPercentPadding = 0;
+		let saasValuePadding = 0;
+
+		// pre-determine the bundle metric environments
+		if (bundle) {
+			for (const metric of Object.values(bundle.metrics)) {
+				metric.envs = Object.entries(metric.envs).map(([ guid, stats ]) => {
+					bundleValuePadding = Math.max(bundleValuePadding, String(format(stats.value)).length);
+					return {
+						guid,
+						name: orgEnvs.find(e => e.guid === guid)?.name || guid,
+						...stats
```

Comment:
Any chances that stats might have a guid or name?

Reply:
Good point, better safe than sorry!